docs/migration/migrate_7_0.asciidoc

```diff
@@ -206,3 +206,54 @@ browsers like Internet Explorer 11 do not support CSP, we show them a warning me
 *Impact:* Nothing needs to be done necessarily, but if you don't need to support legacy browsers like IE11, we recommend
 that you set `csp.strict: true` in your kibana.yml to block access to those browsers entirely. If your organization requires
 users to use IE11, you might like to disable the warning entirely with `csp.warnLegacyBrowsers: false` in your kibana.yml.
+
+[float]
+=== Change in API `/api/kibana/management/saved_objects/relationships/{type}/{id}`
```

Comment:
This is an API only intended for use by the management app, I don't think we need documentation for that. Changing the structure of saved objects to store their references in a different location though, that seems worth mentioning. Do you agree @epixa?